sys-utils/nsenter.c

```diff
@@ -119,6 +122,7 @@ static void __attribute__((__noreturn__)) usage(void)
 }
 
 static pid_t namespace_target_pid = 0;
+static int sock_fd = -1;
```

Comment:
It seems this global variable is unnecessary.

Reply:
Yeah, that's what I initially intended to do, but noticed all those *_fd getopt() variables, thought it's a codestyle.
Going to change that back :-)